src/Core/src/Platform/iOS/KeyboardAutoManagerScroll.cs

```diff
@@ -402,25 +399,34 @@ internal static void AdjustPosition()
 
 		// scenario where we go into an editor with the "Next" keyboard button,
 		// but the cursor location on the editor is scrolled above the visible section
-		else if (View is UITextView && cursorRect.Y < viewRectInWindow.GetMinY())
+		else if (View is UITextView && IsKeyboardShowing && cursorRect.Y < viewRectInWindow.GetMinY())
```

Comment:
Please use `View is UITextView or UITextField` instead in order to include 3rd party input controls